repository/RediStick-Connection-Pool-Tests.package/RsRedisConnectionPoolTest.class/instance/setUp.st

```diff
@@ -0,0 +1,4 @@
+running
+setUp
+	self initializeConnectionPool.
+	connectionPool userName: nil password: self goodPassword
```

Comment:
How about adding `connectionPool password: self goodPassword` short-cut method?

Reply:
yes we can do it

Reply:
fix 23db754